packages/e2e/README.md

```diff
@@ -158,7 +158,7 @@ STAKE_POOL_PROVIDER=stub
 STAKE_POOL_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/stake-pool"}'
 ```
 
-> :information_source: Notice that KEY_MANAGEMENT_PARAMS *mnemonic* property is empty, if you leave this empty on the **local network's** e2e tests a new set of random mnemonics will be generated for you, this is the recommended way of setting up e2e tests on this network.
+> :information*source: Notice that KEY_MANAGEMENT_PARAMS \_mnemonic* property is empty, if you leave this empty on the **local network's** e2e tests a new set of random mnemonics will be generated for you, this is the recommended way of setting up e2e tests on this network.
```

Comment:
Looks like a find/replace error?

![image](https://user-images.githubusercontent.com/303881/217798592-db4450c1-5927-471f-a106-7c4932a3c605.png)

Reply:
This is due to some extensions in my VSCode which takes some not requested initiative. Sorry.

29e4a156d should solve this